lib/services/access_request.go

```diff
@@ -1230,7 +1230,7 @@ func (m *RequestValidator) calculateMaxAccessDuration(req types.AccessRequest) (
 
 	maxDuration := maxDurationTime.Sub(req.GetCreationTime())
 
-	// For dry run requests, the max_duration is set to 7 days.
+	// For dry run requests, the max_duration is set to 14 days.
 	// This prevents the time drift that can occur as the value is set on the client side.
 	// TODO(jakule): Replace with MaxAccessDuration that is a duration (5h, 4d etc), and not a point in time.
```

Comment:
I can't comment on the actual line, but a few lines below you have:

```
return 0, trace.BadParameter("max_duration must be less or equal 7 days")
```

This is no longer correct. I'd recomment:

```
return 0, trace.BadParameter("max_duration must be less or equal to %v" maxAccessDuration)
```